### PR TITLE
Create a "No Op" tracking lib version that can be dropped into a production

### PR DESCRIPTION
**Pull request by adewinter**
Effectively disabling our tracker completely, in case of emergencies.

Also upgraded our dependencies to newer version to keep `yarn audit` happy.

The build script now also builds the `no_op` version of the library.  However, there are not currently any deploy scripts for rolling this out.  At present this feels like such an edge case situation we will want to do it by hand, and with extremely great caution, anyway.

### CODE REVIEW

File: webpack.noop.config.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const webpack = require('webpack'); // eslint-disable-line
+const fileName = 'keen-tracking';
+
+module.exports = {
+  mode: 'development',
```
Comment (Detect) [inline webpack.noop.config.js]: curious what this is used for

Comment (adewinter) [reply webpack.noop.config.js]: Good question:  I used the webpack config generator to make most of this and copied some things from the base webpack config.

**Comment by adewinter** [comment]
@Detect there are two reasons for this
1) Changes to `package.json` cause the dist file to change (yes... I know,  just haven't had the priority to "fix" that)
2) I upgraded some packages due to a security warning given by `yarn audit`.  This could potentially cause changes in the dist files as well.

**Comment by adewinter** [comment]
We definitely want tests.  I'll add some once I figure out a good way how.  Right now I'm going to go ahead and merge this because [fires](https://prodperfect.slack.com/archives/CEHT51W3E/p1561671820345500)

**Comment by adewinter** [comment]
DAMNIT FORGOT TO INCREMENT VERSION

**Pull request merged**